Tests/CodegenCLITests/Support/MockLogLevelSetter.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import CodegenCLI
+import ApolloCodegenLib
+
+extension LogLevelSetter {
+  static var mock: LogLevelSetter.Type {
+    MockLogLevelSetter.self
+  }
+}
+
+struct MockLogLevelSetter: LogLevelSetter {
```

Comment:
There is intentionally no stub for `SetLoggingLevel(Bool)` which relies on the concrete extension of `LogLevelSetter` to function correctly.